Tests/js/views/fields/assets/ez-relation-editview-tests.js

```diff
@@ -28,11 +28,16 @@ YUI.add('ez-relation-editview-tests', function (Y) {
                 method: 'toJSON',
                 returns: this.destinationContentToJSON
             });
-            Y.Mock.expect(this.destinationContent, {
-                method: 'get',
-                args: ['contentId'],
-                returns: 45
-            });
+
+            this.destinationContent.get = function (name) {
```

Comment:
Is there a better way to create mock of this method?

Reply:
Something like this:
```
Y.Mock.expect(this.destinationContent, {
    method: 'get',
    args: [Y.Mock.Value.String],
    run: function (key) {
    	var values = {
    		'resources.MainLocation': true,
    		'contentId': 45
    	}
        return values[key];
    }
});
```

Reply:
It works, thx 🍻